core/src/main/kotlin/org/evomaster/core/problem/rest/classifier/GeneralizedLinearModel.kt

```diff
@@ -26,12 +26,20 @@ class GLMOnlineClassifier(
     private var weights: MutableList<Double>? = null
     private var bias: Double = 0.0
 
+    // classifier state
+    private var cp: Int = 0        // correct predictions
+    private var tot: Int = 1       // total requests
+    private var ac: Double = 0.0   // accuracy ratio
```

Comment:
if you are using exactly the same code as in another class, that is good hint for 2 possible solutions: (1) a shared parent class in which the common logic is pushed up; (2) or extract the functionality in a new class instantiated in both places (eg `ModelAccuracy`)